addons/chemical/functions/fnc_init.sqf

```diff
@@ -18,16 +18,19 @@ params ["_unit", ["_isRespawn", true]];
 if (!local _unit) exitWith {};
 
 if (hasinterface) then {
-    [_unit] call FUNC(coughing);
-    [_unit] call FUNC(handleGasMaskDur);
-    [_unit] spawn FUNC(chemDetector);
-    [_unit] spawn FUNC(breathing);
 
     _unit setVariable [QGVAR(timeleft), missionNamespace getVariable [QGVAR(infectionTime), 60], true];
     _unit setVariable [QGVAR(enteredPoison), false, true];
     _unit setVariable [QGVAR(gasmask_durability), 10, true];
     _unit setVariable [QGVAR(gasmask_durability_reset), false, true];
     _unit setVariable [QGVAR(chemDetectorState), true , true];
+    _unit setVariable [QGVAR(airPoisoning), false, true];
+    _unit setVariable [QGVAR(CS), false, true];
+
+    [_unit] call FUNC(coughing);
+    [_unit] call FUNC(handleGasMaskDur);
+    //[_unit] call FUNC(chemDetector);
```

Comment:
I suppose this should be removed altogether when you've linked it in XEH_postInit, right?

Reply:
Yeah it's not called by init anymore but by client event